client/src/components/App/Issue.css

```diff
@@ -2,3 +2,15 @@
   display: flex;
   flex-grow: 4;
 }
+
+.infoTags {
+  position: flex;
+  float: right;
+
+  font-size: 0.7em;
+  background-color: hotpink;
+  color: white;
+  padding: 5px;
+  margin: 2px;
```

Comment:
Would be nice if you convert em and px to rem since we use that elsewhere. 

`border-radius` should remain px as the browsers round of the nearest px anyway.